VitalTrackApp/src/screens/Login.js

```diff
@@ -0,0 +1,88 @@
+import {
+  View,
+  Text,
+  TextInput,
+  TouchableOpacity,
+  StyleSheet,
+} from 'react-native';
+export default function LoginScreen() {
+  return (
+    <View style={styles.container}>
+      <Text style={styles.title}>Welcome Back</Text>
```

Comment:
Preferably having  "VitalTrack" text above welcome

Reply:
Sure I will address this